### PR TITLE
VZ-7514: Add troubleshooting page for MySQL

### PR DESCRIPTION
**Pull request by mgianatagh**
Add troubleshooting page for MySQL

### CODE REVIEW

File: content/en/docs/troubleshooting/troubleshooting-mysql.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+---
+title: "MySQL"
+linkTitle: "MySQL"
+description: "Troubleshoot MySQL issues"
+weight: 5
+draft: false
+---
+
+There are known issues that can occur with `MySQL`.  The `verrazzano-platform-operator` automatically detects each of the issues described below and performs the actions to repair them.  The `verrazzano-platform-operator` will initiate a repair within a few minutes of an issue being detected.
```
Comment (rosemarymarano) [inline content/en/docs/troubleshooting/troubleshooting-mysql.md:9]: `MySQL` -> is there a reason for using code font here?
Also note: Does `verrazzano-platform-operator` need to be in code font or can it be written as Verrazzano platform operator?
-> The `verrazzano-platform-operator` will automatically detect each of the described issues and perform actions to repair them.
-> The operator initiates a repair within a few minutes of detecting an issue.

File: content/en/docs/troubleshooting/troubleshooting-mysql.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+---
+
+There are known issues that can occur with `MySQL`.  The `verrazzano-platform-operator` automatically detects each of the issues described below and performs the actions to repair them.  The `verrazzano-platform-operator` will initiate a repair within a few minutes of an issue being detected.  
+
+The following sections are provided in the event that a manual repair of an issue is required.
+
+### MySQL pod stuck `Terminating`
+A `MySQL` pod may get stuck in a terminating state.  One possible cause is while upgrading the nodes of a Kubernetes cluster.
+
+This is an example of what this condition looks like.  All the pod containers are terminated, but the pod never finishes terminating.
```
Comment (rosemarymarano) [inline content/en/docs/troubleshooting/troubleshooting-mysql.md:16]: -> Here is an example...

File: content/en/docs/troubleshooting/troubleshooting-mysql.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+$ kubectl describe pods -n keycloak -l component=mysqld
+
+# Excerpt from the command output
+Readiness Gates:
+  Type                          Status
+  mysql.oracle.com/configured   False 
+  mysql.oracle.com/ready        True 
+```
+
+The action to repair this issue is to restart the `mysql-operator` pod.
```
Comment (rosemarymarano) [inline content/en/docs/troubleshooting/troubleshooting-mysql.md:42]: ->  The `verrazzano-platform-operator` repairs this issue by restarting...

File: content/en/docs/troubleshooting/troubleshooting-mysql.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+A `MySQL` pod may get stuck in a terminating state.  One possible cause is while upgrading the nodes of a Kubernetes cluster.
+
+This is an example of what this condition looks like.  All the pod containers are terminated, but the pod never finishes terminating.
+```
+$ kubectl get pods -n keycloak -l component=mysqld
+NAME      READY   STATUS        RESTARTS   AGE
+mysql-0   0/3     Terminating   0          60m
+```
+
+The action to repair this issue is to restart the `mysql-operator` pod.
```
[the remark this answers is not in the document]
Comment (mgianatagh) [reply content/en/docs/troubleshooting/troubleshooting-mysql.md]: fixed as you suggested

File: content/en/docs/troubleshooting/troubleshooting-mysql.md (hunk 3 above)
Comment (mgianatagh) [reply content/en/docs/troubleshooting/troubleshooting-mysql.md]: fixed

**Review by rosemarymarano — APPROVED** [review]

**Pull request merged**